src/main/resources/assets/securitycraft/lang/es_es.json

```diff
@@ -0,0 +1,767 @@
+{
+	"block.securitycraft.laser_block": "Bloque láser",
+	"block.securitycraft.laser.name": "Láser",
+	"block.securitycraft.laser_active": "Activar láser",
+	"block.securitycraft.keypad": "Teclado",
+	"block.securitycraft.iron_door_reinforced": "Puerta reforzada",
+	"block.securitycraft.keycard_reader": "Lector de tarjetas",
+	"block.securitycraft.reinforced_iron_trapdoor": "Trampilla de hierro reforzada",
+	"block.securitycraft.inventory_scanner": "Escáner de inventario",
+	"block.securitycraft.inventory_scanner_field.name": "Campo del escáner de inventario",
+	"block.securitycraft.cage_trap": "Trampa de jaula",
+	"block.securitycraft.deactivated_cage_trap": "Trampa de jaula desactivada",
+	"block.securitycraft.portable_radar": "Radar portátil",
+	"block.securitycraft.reinforced_iron_bars": "Barrotes de hierro reforzados",
+	"block.securitycraft.username_logger": "Registrador de usuarios",
+	"block.securitycraft.bogus_lava": "Lava falsa",
+	"block.securitycraft.bogus_water": "Agua falsa",
+	"block.securitycraft.mine": "Mina",
+	"block.securitycraft.dirt_mine": "Tierra con mina",
+	"block.securitycraft.stone_mine": "Piedra con mina",
+	"block.securitycraft.cobblestone_mine": "Roca con mina",
+	"block.securitycraft.diamond_mine": "Mena de diamante con mina",
+	"block.securitycraft.sand_mine": "Arena con mina",
+	"block.securitycraft.furnace_mine": "Horno con mina",
+	"block.securitycraft.bouncing_betty": "Mina antipersona",
```

Comment:
I disagree in this suggestion. In this case the «bouncing betty» is the name used by americans for the antipersonnel mines form germans (also called Mina-S [you can search about bouncing betties/S-mines in Wikipedia]). Regards!

Reply:
I perfectly know what a bouncing Betty is. The issue here is that a bouncing Betty is indeed an anti-personnel mine, but not all anti-personnel mines are bouncing ones, so the translation is inaccurate. Since you mention the Wikipedia article, you may review the part where it says "es la más conocida de los modelos de minas antipersona _saltarinas_" (it's the most famous model of the _bouncing_ anti-personnel mines). Anti-personnel mine could also be applied to the standard mod's mine, or any of its block variants, or the Claymore Mine ([this](https://www.google.com/search?q=mina+antipersona&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjZ0OLYr8jnAhV4AWMBHVd-DTcQ_AUoAXoECA8QAw&biw=1366&bih=635) vs [this](https://www.google.com/search?q=mina+saltarina+-ni%C3%B1era+-nanny&tbm=isch&ved=2ahUKEwj9-e3Pr8jnAhWB4oUKHd9fAVgQ2-cCegQIABAA&oq=mina+saltarina+-ni%C3%B1era+-nanny&gs_l=img.3...2682.3040..3217...0.0..0.174.487.1j3......0....1..gws-wiz-img.lKhe3Ox7kXw&ei=pgZCXv2OJIHFlwTfv4XABQ&bih=635&biw=1366))

@bl4ckscor3 you might need to step into this since at the end, I have no real power over decisions

Reply:
I'm on the side of LorenaGdL here, based on the argumentation.

Reply:
It's true. Anti-personnel can be applied to other mines like the Claymore, I didn't thought on that. I have never seen them called as «saltarinas» before, but you are right, are called as such.
I also would suggest «Mina-S», but having understood your argument, I would agree now.

Reply:
Finally «Mina saltarina» will be used.